server/graphql/common/features.ts

```diff
@@ -191,6 +191,15 @@ export const getFeatureStatusResolver =
           ? FEATURE_STATUS.ACTIVE // TODO: This flag is misused, there's a confusion between ACTIVE and AVAILABLE
           : FEATURE_STATUS.DISABLED;
       }
+      case FEATURE.PAYPAL_PAYOUTS: {
+        const hasConnectedAccount = await models.ConnectedAccount.count({
+          where: { service: 'paypal', CollectiveId: collective.id },
+          limit: 1,
+        });
+        return hasFeature(collective, FEATURE.PAYPAL_PAYOUTS) && hasConnectedAccount
+          ? FEATURE_STATUS.ACTIVE
+          : FEATURE_STATUS.DISABLED;
```

Comment:
`hasFeature` is already checked in https://github.com/opencollective/opencollective-api/blob/f75c268e52270d11c941222c3f30e790f657fe0f/server/graphql/common/features.ts#L114, I don't think we need to do it again here

```suggestion
        const hasConnectedAccount = await models.ConnectedAccount.count({
          where: { service: 'paypal', CollectiveId: collective.id },
          limit: 1,
        });
        return hasConnectedAccount ? FEATURE_STATUS.ACTIVE : FEATURE_STATUS.DISABLED;
```